plugins/ingestion-kafka/build.gradle

```diff
@@ -0,0 +1,69 @@
+/*
+* SPDX-License-Identifier: Apache-2.0
+*
+* The OpenSearch Contributors require contributions made to
+* this file be licensed under the Apache-2.0 license or a
+* compatible open source license.
+*
+* Modifications Copyright OpenSearch Contributors. See
+* GitHub history for details.
+*/
+
+apply plugin: 'opensearch.internal-cluster-test'
+
+opensearchplugin {
+  description 'Pull-based ingestion plugin to consume from Kafka'
+  classname 'org.opensearch.plugin.kafka.KafkaPlugin'
+}
+
+versions << [
+  'kafka': '2.8.2',
```

Comment:
This looks quite old (September 2022 according to https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients). Why not use the newest available?

Reply:
Good catch. I set it to test with Uber internal kafka, which we are using 2.8.2. Let me upgrade it